docs/DeveloperGuide.adoc

```diff
@@ -96,22 +96,22 @@ When you are ready to start coding,
 [[Design-Architecture]]
 === Architecture
 
-.Architecture Diagram
-image::Architecture.png[width="600"]
+The *_Architecture Diagram_* (Refer to Figure 1) explains the high-level design of the App. Given below is a quick overview of each component.
 
-The *_Architecture Diagram_* given above explains the high-level design of the App. Given below is a quick overview of each component.
+image::Architecture.png[width="600"]
+_Figure 1. Architecture Diagram_
 
 [TIP]
-The `.pptx` files used to create diagrams in this document can be found in the link:{repoURL}/docs/diagrams/[diagrams] folder. To update a diagram, modify the diagram in the pptx file, select the objects of the diagram, and choose `Save as picture`.
+The `.pptx` files used to create diagrams in this document can be found in the link:{repoURL}/docs/diagrams/[diagrams] folder. To update a diagram, modify the diagram in the .pptx file, select the objects of the diagram, and choose `Save as picture`.
 
-`Main` has only one class called link:{repoURL}/src/main/java/seedu/address/MainApp.java[`MainApp`]. It is responsible for,
+`Main` has only one class called link:{repoURL}/src/main/java/seedu/club/MainApp.java[`MainApp`]. It is responsible for the following:
```

Comment:
Should we number `Main`, `Commons` and the rest to make it clear that they are the components? I feel that the image and tip above breaks the flow :/

Reply:
I suppose the flow can be maintained if the tip is relocated